package.json

```diff
@@ -13,6 +13,7 @@
     "react": "16.8.3",
     "react-dom": "16.8.3",
     "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
+    "react-native-button": "^2.4.0",
```

Comment:
@snowmanstark thanks for the contribution. but do we really need a new package "react-native-button" when we could have used <TouchableOpacity /> enclosing <Button /> or <Text />

Reply:
When using `<TouchableOpacity />` the button had to be in a container, and using `<TouchableOpacity />` we would have to tinker around setting the buttons in every page wherever we use buttons.

Reply:
okay looks good thanks.